src/entity/sources/SourceCategory.ts

```diff
@@ -22,4 +23,16 @@ export class SourceCategory {
 
   @UpdateDateColumn()
   updatedAt: Date;
+
+  @Column({
+    type: 'text',
+    update: false,
+    insert: false,
+    nullable: false,
+    unique: true,
+    generatedType: 'STORED',
+    asExpression: `trim(BOTH '-' FROM regexp_replace(lower(trim(COALESCE(LEFT(source_category.title,100),'')||'-'||source_category.id)), '[^a-z0-9-]+', '-', 'gi'))`,
```

Comment:
You should remove it here as well :)

Reply:
Oh, right. Good catch. Let me re-run a migration after deleting just to be sure.